editoast/src/views/rolling_stocks.rs

```diff
@@ -85,6 +89,7 @@ impl From<RollingStockForm> for RollingStockModel {
         RollingStockModel {
             name: Some(rolling_stock.name),
             version: Some(rolling_stock.version),
+            locked: Some(rolling_stock.locked.unwrap_or(false)),
```

Comment:
I don't really know what I did, but i'm pretty sure i put the locked as a query params in the infra model in my other pr, instead of here, on the rolling stock. I'll take care of that.